### ISSUE TITLE
Create a methodology for releasing and updating .js and .min.js without npm

### ISSUE THREAD

**Issue by qq99** [Shopify/twine#28]
Since this project already requires `bundle`, and is a ruby gem, there should be a way to build the `js` and `min.js` files without needing to use `npm` tools

cc @tmlayton

**Comment by pushrax** [comment]
Doesn't really seem too important, most JS devs will have npm anyway.

**Comment by qq99** [comment]
Yeah I'd rate it pretty low priority.  Maybe just adding more stuff to README on what to `npm install` vs `npm install -g`
